lib/pure/collections/tables.nim

```diff
@@ -2342,6 +2336,27 @@ proc inc*[A](t: var CountTable[A], key: A, val = 1) =
     if val != 0:
       insertImpl()
 
+
+proc inc*[A](t: var CountTable[A], key: A, val = 1) =
+  ## Increments `t[key]` by `val` (default: 1).
+  runnableExamples:
+    var a = toCountTable("aab")
+    a.inc('a')
+    a.inc('b', 10)
+    doAssert a == toCountTable("aaabbbbbbbbbbb")
+  
+  incTempl(t, key, val)
+
+proc dec*[A](t: var CountTable[A], key: A, val = 1) =
+  ## Decrements `t[key]` by `val` (default: 1).
+  runnableExamples:
+    var a = toCountTable("aaabbbbbbbbbbb")
+    a.dec('a')
+    a.dec('b', 10)
+    doAssert a == toCountTable("aab")
+
+  incTempl(t, key, -val)
```

Comment:
This breaks if `val` is `int.low` (in fact, pretty much every use of `-` without checking if the value is `T.low` probably breaks...), it would be better to just copy the `inc` implementation imo. While we're at it, also add a test for that.